parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/ColumnChunkMetaData.java

```diff
@@ -380,13 +382,15 @@ public PrimitiveType getPrimitiveType() {
   /**
    * @return the stats for this column
    */
+  @JsonIgnore
```

Comment:
Does this ignore printing stats for all columns?

Reply:
yes, with that annotation, all stats are skipped from logging. That's a behavior change.

I did so to align behavior with the newly added `SizeStatistics`. As the default implementation throws an exception, we can't safely serialize this field so I added `JsonIgnore` annotation there too. (I'm actually wondering why `null` is not returned as default).

If we decide to serialize the `SizeStatistics`, I'm also wondering how we should handle [invalid stats](https://github.com/apache/parquet-mr/blob/f12229addfdca512770a0959d8074100b1e42ee5/parquet-column/src/main/java/org/apache/parquet/column/statistics/SizeStatistics.java#L49).

Reply:
Although it is used in the logging, I'm not sure if ignoring the stats silently will confuse/break developers used to depend on that log to debug the stats.

@gszadovszky @Fokko Do you have any suggestion?

Reply:
I think we're good here. I'm not sure if anyone uses the stats for debugging since it is super verbose.